code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -289,7 +289,7 @@
 	breath_heat_level_2 = 500	//Default 450
 	breath_heat_level_3 = 1350	//Default 1250
 
-	reagent_tag = IS_SKRELL
+	//reagent_tag = IS_SKRELL //VOREStation Edit
```

Comment:
Mlem. Just do this in the _vr file too. Just make it whatever the default is in species.dm.